synapse/daemon.py

```diff
@@ -106,7 +106,7 @@ def _onDmonFini(self):
 
         for celldir, proc in list(self.cellprocs.items()):
             proc.terminate()
-            proc.join(2)
+            proc.join(30)
```

Comment:
Is this due to slow proc termination on OSX (because of cleaning up non-sparse LMDB files)?

Reply:
Yes